crates/languages/src/rust/highlights.scm

```diff
@@ -147,20 +150,16 @@
   "&&"
   "*"
   "*="
-  "*"
   "+"
   "+="
-  ","
   "-"
   "-="
   "->"
-  "."
   ".."
   "..="
   "..."
   "/="
   ":"
-  ";"
```

Comment:
Nice. So now `;`, `.` and `,` are just `punctuation.delimiter` and not `operator` which matches how other languages do it.

Good catch of the duplicated `*`.